components/browser/session/src/test/resources/manifests/spec_typical.json

```diff
@@ -0,0 +1,38 @@
+{
+  "lang": "en",
+  "dir": "ltr",
+  "name": "Super Racer 3000",
+  "description": "The ultimate futuristic racing game from the future!",
+  "short_name": "Racer3K",
+  "icons": [{
+    "src": "icon/lowres.webp",
+    "sizes": "64x64",
+    "type": "image/webp"
+  },{
+    "src": "icon/lowres.png",
+    "sizes": "64x64"
+  }, {
+    "src": "icon/hd_hi",
+    "sizes": "128x128"
```

Comment:
Just for the sake of having more use cases tested, could we add a sample with an icon with multiple sizes? None of the manifest samples has it. Something like  ` "sizes": "128x128 192x192"`